process/project_proposals.adoc

```diff
@@ -70,8 +70,13 @@ image::incubation-process.png[Incubation process]
 
 === Graduation process
 
-Graduation process follows the pattern of incubation, but the majority of DD should have happened at the incubation phase so it should be a less onerous process for graduation.
-
+. *Submit Graduation Proposal Template*
+   * Project fills out and submits the link:graduation-proposal-template.md[graduation proposal template] in a pull request in the https://github.com/cncf/toc[cncf/toc GitHub repo]. The file should be located in https://github.com/cncf/toc/proposals/graduation/[the graduation proposals directory].
+. *Project gets scheduled to present at public TOC meeting*
```

Comment:
I'm wondering if we should skip the presentation step and go straight into the two week public comment period where the SIG can weigh in if they choose. What do we think? @cncf/toc

Reply:
For graduation, that makes a lot of sense to me. I mean, a project at graduation has already done multiple presentations to the TOC, and we've been wondering exactly what is left to be said other than "yes, we did check off all the checkboxes," which is redundant with our written documentation.